Add com.obsproject.Studio.Plugin.OBSVkCapture (#2708)
flathub/flathub: README.md

Vulkan Capture for OBS
======================

> This is a plugin for OBS that allows Vulkan/OpenGL applications to be captured. For instructions on how to use, see [this page](https://github.com/nowrep/obs-vkcapture#usage).

Reviewer: ```suggestion
This is a plugin for OBS Studio that allows Vulkan/OpenGL applications to be captured. For instructions on how to use, see [this page](https://github.com/nowrep/obs-vkcapture#usage).
```

> For this to work you also need the corresponding tools to start the application with (should be org.freedesktop.Platform.VulkanLayer.VkCapture on flathub)

Reviewer: ```suggestion
For this plugin to work you also need the corresponding tools to start the application with. These tools are available [on Flathub as `org.freedesktop.Platform.VulkanLayer.VkCapture`](https://github.com/flathub/org.freedesktop.Platform.VulkanLayer.VkCapture].
```
